server/src/dotenv.config.ts

```diff
@@ -0,0 +1,15 @@
+import { config } from 'dotenv'
+
+config({
+  path: '../.env',
+})
+
```

Comment:
I hope this was left here by mistake :)) 
Always use descriptive function and variable names.